src/components/game/components/board/withBoardProps.js

```diff
@@ -69,15 +81,33 @@ const mapStateToProps = (state) => ({
   playerHand: playerHand(state),
   playerMoney: playerMoney(state),
   pot: pot(state),
+  logger: logger(state),
 });
 
 const mapDispatchToProps = (dispatch) => ({
-  onClickPlayHandler: () => dispatch(startGame()),
-  onClickFoldHandler: () => dispatch(fold()),
-  onClickCheckHandler: () => dispatch(check()),
-  onClickRaiseHandler: () => dispatch(raise()),
-  onClickReplaceHandler: (card) => dispatch(replace(card)),
-  onClickNextPhaseHandler: () => dispatch(nextPhase()),
+  onClickPlayHandler: () => {
+    dispatch(startGame());
+    dispatch(startGameLog());
+  },
+  onClickFoldHandler: () => {
+    dispatch(fold());
+    dispatch(foldLog());
+  },
+  onClickCheckHandler: () => {
+    dispatch(check());
+    dispatch(checkLog());
+  },
+  onClickRaiseHandler: () => {
+    dispatch(raise());
+    dispatch(raiseLog());
+  },
+  onClickReplaceHandler: (card) => {
+    dispatch(replace(card));
+    dispatch(replaceLog());
+  },
+  onClickNextPhaseHandler: () => {
+    dispatch(nextPhase());
+  },
```

Comment:
reducer can listen to actions that are related to other reducers as well for example you can listen to startGame in both game and log reducers and act accordingly

Reply:
Done I think